src/route/plant-router.js

```diff
@@ -58,6 +62,24 @@ plantRouter.put('/plants/:id', bearerAuthMiddleware, jsonParser, (request, respo
     .catch(next);
 });
 
+plantRouter.put('/plants/:id/image', bearerAuthMiddleware, jsonParser, multerUpload.any(), (request, response, next) => {
+  const file = request.files[0];
+  const key = `${file.filename}.${file.originalname}`;
+  const options = { funValidators: true, new: true };
```

Comment:
`funValidators` ! I'm sure this is super fun, but hope this got corrected eventually. :)